### DISCUSSION TITLE
Separation between the p-dropdown and the panel when it opens upwards and there is a filter v16.9.15lts

### DISCUSSION THREAD

**Discussion by RDiazItopia** [primefaces/community#2604] [PrimeNG]
![Screenshot 2024-08-29 143740](https://github.com/user-attachments/assets/44abf714-0976-45da-a5b2-ff867d863832)

**Comment by qwadrox** [comment]
Yes. I could reproduce. https://stackblitz.com/edit/qfu6jh?file=package.json 
I created an issue to track: https://github.com/primefaces/primeng/issues/16318
